src/util/TypeUtil.ts

```diff
@@ -0,0 +1,69 @@
+import { workspace } from "vscode";
+
+/**
+ * Provides helper function to types
+ */
+export default class TypeUtil {
+    /**
+    * Holds the current instance
+    *
+    * @type {TypeUtil}
+    */
+    private static _instance: TypeUtil;
+
+    /**
+     * Holds wether we use short names or not
+     *
+     * @type {bool|null}
+     */
+    private _useShortNames: any;
+
+    private constructor() {
```

Comment:
If you make this public then you can just create an instance for each test instead of relying on the singleton, does enforce the singleton less but I think it's worth it for nicer testing.

I would also consider getting rid of the constructor altogether and create a getter for the useShortNames that when it's null will access the config and set it. This way it only accesses the config if it needs to.